returnn/torch/engine.py

```diff
@@ -52,6 +57,8 @@ def train(self):
     start_epoch, _ = self.get_train_start_epoch_batch(self.config)
     final_epoch = self.config_get_final_epoch(self.config)
 
+    self._load_model(epoch=start_epoch)
```

Comment:
This should not be here but in `init_train_from_config` or `init_network_from_config`.

Reply:
Just like `start_epoch` etc. Maybe I just merge this for now.